server/api/Users.js

```diff
@@ -9,64 +9,74 @@ const {
 } = require('./Utils');
 // add require by token middleware
 
-// GET /api/users/all
-// add requireAdminToken util
+// GET /api/users/all returns all users if given a valid Admin token
 router.get('/all', requireAdminToken, async (req, res, next) => {
   try {
-    res.send(await User.findAll());
+    res.send(await User.findAll({ attributes: { exclude: ['password'] } }));
   } catch (er) {
     next(er);
   }
 });
```

Comment:
Even hashed, I think we want to avoid sending our users passwords back to the front-end.